pkg/controller/common.go

```diff
@@ -447,18 +451,6 @@ func (c *SharedState) getPodsFromVolume(volumeName string) ([]string, error) {
 	return pods, nil
 }
 
-func (c *SharedState) getVolumesForPods(pods ...string) ([]string, error) {
-	volumes := []string{}
-	for _, pod := range pods {
-		podVolumes, err := c.getVolumesFromPod(pod)
-		if err != nil {
-			return nil, err
-		}
-		volumes = append(volumes, podVolumes...)
-	}
-	return volumes, nil
-}
-
 func (c *SharedState) getVolumesForPodObjs(pods ...v1.Pod) ([]string, error) {
```

Comment:
`getVolumesForPodObjs` should really take `[]v1.Pod` rather than variadic arguments.

Reply:
Changed to []v1.Pod